treeherder/webapp/api/performance_data.py

```diff
@@ -814,6 +815,15 @@ def list(self, request):
                 base_stddev_pct = self._get_stddev_pct(base_avg_value, base_stddev)
                 new_stddev_pct = self._get_stddev_pct(new_avg_value, new_stddev)
 
+                params = {
```

Comment:
You need 

- `highlightedRevisions `for base 
- `highlightedRevisions` for new
- `series `which is the repository
- [signature hash](https://github.com/mozilla/treeherder/blob/dccb26f0be4f04b483724d7d9a698d2e9fe7b69d/ui/perfherder/compare/CompareView.jsx#L113) 
- `timerange `which is the interval, and you already have access to it 
([Example](https://treeherder.mozilla.org/perfherder/graphs?highlightedRevisions=9d777ac6ab3a&highlightedRevisions=e651649b0dac&series=mozilla-central%2C7c6f5484d91eb6a722e841c45ed9411f9cf830e0%2C1%2C1&timerange=172800))